README.md

```diff
@@ -1 +1,4 @@
 # testingspace
+
+### Hello, this is an edit.
+This is a second edit.
```

Comment:
```suggestion
### Hello, this is an edit.
This is a second edit containing more detials and inflromatoin.
```

Add extra info

Reply:
```suggestion
### Hello, this is an edit.
This is a second edit containing more details and information.
```
Corrected a couple of typos.

Reply:
🤦DOH!  Well spotted. 

Happy with your corrections

Reply:
No problem, thanks.